tests/unit/contentproviders/test_figshare.py

```diff
@@ -50,10 +50,16 @@ def test_content_id(link, expected):
     (
         "https://doi.org/10.6084/m9.figshare.9782777.v1",
         {"host": test_fig.hosts[0], "article": "9782777", "version": "1"},
+        # $ curl -sIL https://doi.org/10.6084/m9.figshare.9782777.v1 | grep location
+        # location: https://figshare.com/articles/Binder-ready_openSenseMap_Analysis/9782777/1
+        # location: https://figshare.com/articles/code/Binder-ready_openSenseMap_Analysis/9782777
```

Comment:
I presume this must be a Figshare bug (`.v1` -> `/1` -> missing)? If someone can confirm I'll send them an email.

Reply:
I don't think we have anyone who knows/uses figshare enough to know the answer. So I'd reach out to them.

(We aren't using their API though, I think. We are getting ourselves redirected from a doi.org URL and then parsing the figshare.com URL we end up at)